src/main/webapp/setup.jsp

```diff
@@ -93,23 +93,23 @@ if (Setup.isInstalled()) {
 								<label for="dbhost"><fmt:message key="generic.text.setup.host" /></label>
 							</div>
 							<div class="col-75">
-								<input type="text" id="dbhost" name="dbhost" placeholder="Database Hostname..." autofocus required>
+								<input type="text" id="dbhost" name="dbhost" value="localhost" autofocus required>
 							</div>
 						</div>
 						<div class="row">
 							<div class="col-25">
 								<label for="dbport"><fmt:message key="generic.text.setup.port" /></label>
 							</div>
 							<div class="col-75">
-								<input type="text" id="dbport" name="dbport" placeholder="Database Port..." required>
+								<input type="text" id="dbport" name="dbport" value="3306" required>
 							</div>
 						</div>
 						<div class="row">
 							<div class="col-25">
 								<label for="dbuser"><fmt:message key="generic.text.setup.user" /></label>
 							</div>
 							<div class="col-75">
-								<input type="text" id="dbuser" name="dbuser" placeholder="Shepherd Database User.." required>
+								<input type="text" id="dbuser" name="dbuser" value="root" required>
```

Comment:
This needs to be changed back to placeholders

Reply:
To be clear, don't mind the default values being there, but they should be placeholders not values

Reply:
> @etnoy Is this still WIP or ready to merge?

No, not WIP anymore. I would say so if you're alright with the above comments not being fixed yet. Thanks for taking a look at my pull request.